nemo/collections/nlp/data/language_modeling/megatron/_megatron_batch_samplers.py

```diff
@@ -0,0 +1,197 @@
+import abc
```

Comment:
Can you please copy paste the copyright from another file in nemo to this file?

Reply:
Also, can you rename this file to `megatron_batch_samplers.py` We don't have filenames starting with `_` in nemo.